src/main/resources/application.yml

```diff
@@ -9,7 +9,7 @@ spring:
     password: ${DB_PASSWORD:password}
     username: ${DB_USERNAME:sa}
     url: ${DB_URL:jdbc:h2:mem:testdb}
-  h2.console.enabled: false
+  h2.console.enabled: true
```

Comment:
The console should only be enabled locally.

Reply:
Ah. I enabled to see the database tables and forgot to disable it. 
Noted